glancesync/glancesync.py

```diff
@@ -415,12 +416,18 @@ def init_logs(include_date=False):
         :return:
         """
 
+        '''
+        We do not need to create a new logger just import it from app.settings.log.py
+        TO BE CHECKED
+
```

Comment:
Be very careful with this change and what it implies.

This method is used by the cmdline tool, to show the info messages about the progress by console. If you modify this, you should include a default configuration with exactly the same behaviour.

The point of this method is that you may need to change the logging in a program (e.g. the web server that provide the API) via configuration but without affecting the behaviour of the cmdline utils, via a specific configuration hardcoded in a method to the cmdline tool.
I think that the changes cannot add complexity to the configuration of the glancesync cmdline tool. Nowadays, it works out of the box without configuration.